_layouts/index.html

```diff
@@ -112,6 +119,50 @@ <h2 id="question">{{ t.pagecontent.question }}</h2>
       </div>
     </li>
 
+    <li>
+      <div class="group row">
+        <div class="col-1">
+          <p>“To install, drag this icon…” no more! <code>brew cask</code> brings Homebrew's elegance, simplicity and speed to macOS applications and large binaries alike.</p>
```

Comment:
Apostrophe in `Homebrew’s` should be `’` instead of `'` (the homepage already in place has the correct usage).

Reply:
This is done by GitHub Pages. I'll remove other "smart quotes" in here.